ext/oci8/oci8.stub.php

```diff
@@ -226,14 +226,6 @@ function oci_field_is_null($statement_resource, string|int $column_number_or_nam
  */
 function ocicolumnisnull($statement_resource, string|int $column_number_or_name): bool {}
 
-function oci_internal_debug(bool $mode): void {}
```

Comment:
The `PHP_FUNCTION` declaration itself should also be removed. + maybe a note in the UPGRADING file can be added too (although `oci_internal_debug()` is probably not a very actively used function).

Reply:
@kocsismate I did that, but i think it got lost while rebasing. Will fix that asap.

Reply:
@kocsismate fixed.